containers/ecr-viewer/src/app/components/NavLinks.tsx

```diff
@@ -0,0 +1,40 @@
+import Link from "next/link";
+
+import { getLoggedInUser, isAdmin } from "@/app/services/userService";
+import { getLoggedInUserSession } from "@/app/utils/auth-utils";
+
+import UserMenu from "./UserMenu";
+
+/**
+ * Nav links component for the eCR Viewer Header.
+ * This component renders navigation links for admin users and returns a
+ * user menu containing user information and a sign-out button
+ * @returns The header nav links and a user menu
+ */
+const NavLinks = async () => {
+  const sessionUser = await getLoggedInUserSession();
+  if (!sessionUser) return;
+
+  const dbUser = await getLoggedInUser();
+
+  return (
+    <div className="display-flex flex-row">
+      {isAdmin(dbUser) && (
+        <div className="display-flex flex-row">
```

Comment:
given the outer part is a div with the same classes can we change this div to a fragment (`<>`) and skip the div?